### PR TITLE
Cleaning

### PR DESCRIPTION
**Pull request by bethanyios**
Old folders/files have been removed now, and anything potentially relevant as been moved into the new structure - those will need to be edited with improved content

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,36 +8,55 @@ description: >-
...
-
-The Product Management Curriculum was validated as part of Collaborative Digital Training (May - June 2020).
+The Product Management Curriculum was validated as part of **Collaborative Digital Training** (May - June 2020) and the **Discovery Learning Programme** (Sept - Oct 2020).
 
 ## Learning Outcomes
 
 - Understand the product development cycle from discovery and definition to build phase
 - Understand the role and responsibilities of a Product Manager and a product owner
-- Understand Scrum and Agile
 - Learn methods and techniques that facilitate user-centred, consensus-driven decision making in Definition, Discovery and Build phase
```
Comment (Albadylic) [inline README.md:17]:
```suggestion
- Learn methods and techniques that facilitate a user-centred approach to Discovery, Definition, Prototyping and Building. 
```

I know you didn't write this (it was me lol) but could we say this instead?

File: list-of-exercises/user-personas.md [hunk 2]
```diff
@@ -8,25 +8,25 @@ description: >-
 
 ### Introduction
 
-User Personas are fictional characters that have a name and picture; relevant characteristics such as a role, activities, behaviours, and attitudes; and a goal, which is the problem that has to be addressed or the benefit that should be provided. 
+User Personas are fictional characters that have a name and picture; relevant characteristics such as a role, activities, behaviours, and attitudes; and a goal, which is the problem that has to be addressed or the benefit that should be provided.
```
[the remark this answers is not in the document]
Comment (Albadylic) [reply list-of-exercises/user-personas.md:11]: "fictional representations of groups of users"

File: mentor-notes.md [hunk 3]
```diff
@@ -1,48 +0,0 @@
...
-
-In larger cohorts, communication should be handled via Slack (or a similar instant messaging platform). This provides quick and clear communication that is advantageous to communicating via email.
-
-Participants should be informed about the agenda of each workshop in advance so that they know what to expect and what to prepapre.
-
-## Feedback
-
-During the programme, organisations should be given opportunities to give feedback to facilitators. This aids mutual understanding and allows the programme's pace to be adjusted to the pace of participants. The last 15 minutes of each workshop should be dedicated to questions and check-out.
-
-This also encourages participants to act as a cohort and to feel in charge of the process.
```
Comment (Albadylic) [inline mentor-notes.md:48]: This file could be the basis of the top-level facilitator notes - with a decent overhaul.

**Review by Albadylic — APPROVED** [review]
Think there's one phrasing change I suggested and one file which could be kept - otherwise, happy for this to be merged. 

And sorry for lamenting some of the old copy - I got a little distracted...

File: README.md [hunk 4]
```diff
@@ -8,36 +8,55 @@ description: >-
...
 
 ## Goals
 
 - Improve the skills of Product Managers
-- Build the capacity of a charitable organisations \(and the charity sector\).
+- Build the capacity of a charitable organisation \(and the charity sector\)
 - Foster collaboration between charitable organisations and Product Managers
 
-## Content
+## How to use this book:
```
[the remark this answers is not in the document]
Comment (bethanyios) [reply README.md:26]: yay!

File: README.md (hunk 1 above)
Comment (bethanyios) [reply README.md:17]: of course!

**Review by jc2820 — APPROVED** [review]

File: list-of-exercises/user-personas.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (bethanyios) [reply list-of-exercises/user-personas.md:11]: haha will change more of the copy in this doc as we discuss and will defo add that change

File: mentor-notes.md (hunk 3 above)
Comment (bethanyios) [reply mentor-notes.md:48]: Wanted to discuss this with you. I have made a space in readme for facilitator notes as agreed but maybe we want a whole page for it - these notes are saved on my comp so we can add best bits in either way

**Pull request merged**